### ISSUE TITLE
epoptes server  does not start

### ISSUE THREAD

**Issue by XanthiM** [Epoptes/epoptes#75]
Hello. I have installed epoptes in  standalone  mode. I have been using it with no problem. But today, after some updates made by mint 19, it stoped running. 
The error report is

**Traceback (most recent call last):
  File "/usr/bin/epoptes", line 99, in <module>
    main()
  File "/usr/bin/epoptes", line 89, in main
    egui = gui.EpoptesGui()
  File "/usr/lib/python3/dist-packages/epoptes/ui/gui.py", line 83, in __init__
    self.get('mi_remote_assistance').set_property('visible', False)
AttributeError: 'NoneType' object has no attribute 'set_property'**

After that it will not start.
What can i do?

**Comment by alkisg** [comment]
Thanks, fix committed in https://github.com/Epoptes/epoptes/commit/32481b49e3502e546c0349cce258eb5bedb40116.

The problem was caused by a bad menu item name. But it only showed in your installation because LinuxMint violates the Debian Policy by not installing the recommended packages by default:

> https://www.debian.org/doc/debian-policy/ch-relationships.html
> The Recommends field should list packages that would be found together with this one in all but unusual installations.

So when you install things in LinuxMint, remember to always add --install-recommends to avoid this LinuxMint problem. To fix this bug in Epoptes now run:
```
apt remove epoptes
apt install --install-recommends epoptes
```

**Comment by XanthiM** [comment]
Ευχαριστώ πάρα πολύ για την άμεση απάντηση.

Το πρόβλημα λύθηκε.


________________________________
Από: Alkis Georgopoulos <notifications@github.com>
Στάλθηκε: Τετάρτη, 26 Σεπτεμβρίου 2018 11:50 πμ
Προς: Epoptes/epoptes
Κοιν.: XanthiM; Author
Θέμα: Re: [Epoptes/epoptes] epoptes server does not start (#75)


Thanks, fix committed in 32481b4<https://github.com/Epoptes/epoptes/commit/32481b49e3502e546c0349cce258eb5bedb40116>.

The problem was caused by a bad menu item name. But it only showed in your installation because LinuxMint violates the Debian Policy by not installing the recommended packages by default:

https://www.debian.org/doc/debian-policy/ch-relationships.html
The Recommends field should list packages that would be found together with this one in all but unusual installations.

So when you install things in LinuxMint, remember to always add --install-recommends to avoid this LinuxMint problem. To fix this bug in Epoptes now run:

apt remove epoptes
apt install --install-recommends epoptes
